lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -227,8 +220,18 @@ spec = do
             , "  root                     Extract root extended private key from"
             , "                           a mnemonic sentence."
             , "  child                    Derive child keys."
-            , "  public                   Extract public key from a private key."
+            , "  public                   Extract the public key from a private"
+            , "                           key."
             , "  inspect                  Show information about a key."
+            , ""
+            , "Keys are read from standard input for convenient chaining of commands."
```

Comment:
Thought it looked weird above the `Usage`, so I moved it to the footer.

Reply:
Yep, this is better.